app/src/pages/HomePage/HomePage.tsx

```diff
@@ -32,13 +42,27 @@ export const HomePage = (): React.ReactElement => {
   }, [contract, requester]);
 
   React.useEffect((): void => {
-    if (!contract) {
+    if (!contract || chainId != ChainId.Rinkeby) { 
       setShowBrowserError(true);
```

Comment:
then here you can have separate branches and set the error text to the thing you want it to be